internal/storage/v2/grpc/README.md

```diff
@@ -0,0 +1,14 @@
+# gRPC Remote Storage
```

Comment:
this is pretty short, should we add more content from https://github.com/jaegertracing/jaeger/tree/main/internal/storage/v1/grpc#certifying-compliance?

Reply:
@yurishkuro Should we build out the test binary first before adding those instructions?

Reply:
no, for now instructions can suggest to clone jaeger repo and run `go test xyz` to execute the grpc harness